website/settings/defaults.py

```diff
@@ -174,6 +176,15 @@ def parent_dir(path):
 ## Default RabbitMQ broker
 BROKER_URL = 'amqp://'
 
+CELERY_CREATE_MISSING_QUEUES = False
+CELERY_DEFAULT_QUEUE = 'osf'
+CELERY_QUEUES = (
```

Comment:
Since this appears to only be an issue for dev environments, why not add it to local-dist.py instead? That would work around the issues of not being able to import settings in `tasks.py`, wouldn't it?